ChangeLog

```diff
@@ -1,3 +1,6 @@
+Version 7.8.1
```

Comment:
FYI I've not done anything with 7.8.0 yet, so you could just roll it into that version.